lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -339,15 +343,29 @@ defmodule ExDoc.Formatter.HTML.Autolink do
   or trailing `]`, e.g. `[my link link/1 is here](url)`, the fun/arity
   will get translated to the new href of the function.
   """
-  def local_doc(bin, locals) when is_binary(bin) do
+  def local_doc(bin, locals, extension \\ ".html", lib_dirs \\ elixir_lib_dirs()) when is_binary(bin) do
```

Comment:
Kernel.SpecialForms calls are neither local as in `local_doc` nor remote as in `elixir_functions` calls, so having this logic doesn't quite fit either.

Another approach is to combine `local_doc` and `elixir_functions` into one function, see: https://github.com/elixir-lang/ex_doc/blob/wm-autolink-bak/lib/ex_doc/formatter/html/autolink.ex#L347